src/mongodb.ts

```diff
@@ -23,7 +23,7 @@ const path = `mongodb://${user}:${password}@${address}/${db}`
 
 export const setupMongoConnection = async () => {
   // FIXME we may not want to keep this in the logs
-  baseLogger.info({path}, "db connection path")
+  baseLogger.info({ path }, "db connection path")
```

Comment:
I think we should remove this now. It outputs the password in the logs.

Reply:
unrelated to this PR but I've push the fix here.

Reply:
Yep, I just suggested because this PR and the requested change were pretty small